### PR TITLE
[Docs] Removed occurences of Ubuntu 16.04 from documentation.

### PR DESCRIPTION
**Pull request by RRathna**
Signed-off-by: Rathna Ramesh <rathna1993@gmail.com>

### CODE REVIEW

File: docs/GettingStartedDocs/VisualStudioLinux.md [hunk 1]
```diff
@@ -36,12 +36,10 @@ but a non-SGX machine can still be used:
 
 On the Linux build machine, or after opening an ssh session into the VM:
 
-- Install the Open Enclave SDK.  See [installation instructions for Ubuntu 16.04](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/install_oe_sdk-Ubuntu_16.04.md)
-  or [installation instructions for Ubuntu 18.04](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/install_oe_sdk-Ubuntu_18.04.md), except that step 2 on those pages is outdated and result
+- Install the Open Enclave SDK.  See [installation instructions for Ubuntu 18.04](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/install_oe_sdk-Ubuntu_18.04.md), except that step 2 on those pages is outdated and result
   in SGX not working.  Instead, replace step 2 with the
```
Comment (RRathna) [inline docs/GettingStartedDocs/VisualStudioLinux.md:40]: Hi @radhikaj, are the instructions in lines 40-42 still relavent?

File: docs/GettingStartedDocs/README.md [hunk 2]
```diff
@@ -33,18 +33,12 @@ the hardware types below.
 
 ### Intel SGX
 
-If you would like to start developing apps with the preview Open Enclave SDK
-release, start here for instructions to install and use the SDK package:
-
-- [Ubuntu 18.04 with SGX hardware](install_oe_sdk-Ubuntu_18.04.md)
```
Comment (mingweishih) [inline docs/GettingStartedDocs/README.md:39]: Why this is removed?

File: docs/GettingStartedDocs/README.md [hunk 3]
```diff
@@ -33,18 +33,12 @@ the hardware types below.
...
-- [Ubuntu 16.04 with SGX hardware](install_oe_sdk-Ubuntu_16.04.md)
-- [Ubuntu 18.04 or 16.04 in simulation mode](install_oe_sdk-Simulation.md)
```
Comment (mingweishih) [inline docs/GettingStartedDocs/README.md:41]: Should we just remove the 16.04 word there?

File: docs/GettingStartedDocs/README.md [hunk 4]
```diff
@@ -33,18 +33,12 @@ the hardware types below.
...
-- [Windows 10(1709 or higher), Windows Server 2016 or Windows Server 2019 on an Intel machine](install_oe_sdk-Windows.md)
```
Comment (mingweishih) [inline docs/GettingStartedDocs/README.md:42]: why is this removed?

File: docs/GettingStartedDocs/README.md (hunk 2 above)
Comment (RRathna) [reply docs/GettingStartedDocs/README.md:39]: @radhikaj suggested that this be removed since these documentations are shipped bundled with the releases.

File: docs/GettingStartedDocs/VisualStudioLinux.md (hunk 1 above)
Comment (radhikaj) [reply docs/GettingStartedDocs/VisualStudioLinux.md:40]: @rs-- Can you please help us understand if 40-42 are still relevant?

File: docs/GettingStartedDocs/README.md (hunk 2 above)
Comment (radhikaj) [reply docs/GettingStartedDocs/README.md:39]: @RRathna, I think this can stay. Since it relates to 1804.

File: docs/GettingStartedDocs/README.md (hunk 3 above)
Comment (radhikaj) [reply docs/GettingStartedDocs/README.md:41]: @RRathna, I am sorry for any confusion, but the 1804 and simulation mode instructions can stay

Comment (radhikaj) [reply docs/GettingStartedDocs/README.md:41]: This is because someone who looks at our master branch should understand how to install OE SDK .

File: docs/GettingStartedDocs/README.md (hunk 4 above)
Comment (radhikaj) [reply docs/GettingStartedDocs/README.md:42]: @RRathna , this can stay too.

File: docs/GettingStartedDocs/VisualStudioLinux.md (hunk 1 above)
Comment (rs--) [reply docs/GettingStartedDocs/VisualStudioLinux.md:40]: You can remove: "except that step 2 on those pages is outdated and result in SGX not working. Instead, replace step 2 with the SGX1 instructions prior to the Install section."

File: docs/GettingStartedDocs/README.md (hunk 3 above)
Comment (RRathna) [reply docs/GettingStartedDocs/README.md:41]: Adding back steps to install OE SDK

File: docs/GettingStartedDocs/README.md (hunk 2 above)
Comment (RRathna) [reply docs/GettingStartedDocs/README.md:39]: Adding back steps to install OE SDK.

**Review by radhikaj — APPROVED** [review]
Thanks @RRathna

**Review by rs-- — APPROVED** [review]
